TESTING.md

```diff
@@ -0,0 +1,76 @@
+# Testing Tracetest
+
+On Tracetest, we work in two ways to test the entire system and guarantee that everything is working fine. One is through automatic tests, where we run unit, integration and end-to-end tests on Tracetest CLI, Web UI and API server.
+
+Another source of tests is the manual tests that we execute on each release, simulating users, and checking if everything is ok on these tests.
+
+## Manual Tests
+
+On our manual tests, we aim to do some [sanity checks](https://en.wikipedia.org/wiki/Sanity_check) to see if the main features are working as expected. Usually, we run it on each Tracetest release.
+
+### Testing Tracetest setup
```

Comment:
Things to think in the future: this is a good candidate to become a cypress test for each environment, so we can validate a complete setup flow.